drivers/osdp/src/osdp_phy.c

```diff
@@ -0,0 +1,279 @@
+/*
+ * Copyright (c) 2020 Siddharth Chandrasekaran <siddharth@embedjournal.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <logging/log.h>
+LOG_MODULE_DECLARE(osdp, CONFIG_OSDP_LOG_LEVEL);
+
+#include <string.h>
+#include "osdp_common.h"
+
+#define TAG "PHY: "
+
+#define PKT_CONTROL_SQN			0x03
+#define PKT_CONTROL_CRC			0x04
+#define PKT_CONTROL_SCB			0x08
+
+struct osdp_packet_header {
```

Comment:
This should be `__packed__` if it goes over the line

Reply:
Done.